Tasks/NuGetInstaller/NuGetInstaller.ps1

```diff
@@ -8,6 +9,8 @@ param(
     [string]$nuGetPath
 )
 
+. .\VsoNuGetHelper.ps1
```

Comment:
All messages for Write-Host, throw, Write-Error, and Write-Warning should use the Get-LocalizedString cmdlet.

You'll need to move this statement to dot source the helper functions down below the module import statements. The helper functions you are importing have some error strings. Those should be retrieved using the localization helper cmdlet, which is available only after importing the below modules.